code/modules/reagents/reactions/instant/vox.dm

```diff
@@ -0,0 +1,67 @@
+/decl/chemical_reaction/protoslurry
```

Comment:
Do these recipes actually get instantiated? As I recall, I repathed the regular reactions as subtypes of /instant either to organizationally differentiate them from distillery recipes, _or_ because distillery recipes were causing an issue and doing so was prudent.